astropy/utils/tests/test_data.py

```diff
@@ -1000,12 +1000,15 @@ def test_is_url_in_cache_local(temp_cache, valid_urls, invalid_urls):
     assert not is_url_in_cache(nonexistent)
 
 
+@pytest.mark.filterwarnings('ignore:Not clearing data cache - '
```

Comment:
@aarchiba , if you are okay with it, we can ignore this warning for now and then open a follow-up issue to deal with it post-4.0?

Reply:
How tight are we for time? I can probably get this straightened out in a day or two but I don't want to delay anything.

Reply:
Not *that* tight! 😅  Thanks!

Reply:
I approve of the quest to banish all warnings! PINT emits so many warnings users can't reliably tell when they're losing accuracy.

Reply:
I think I fixed this one.

Reply:
Yup. Thanks!